cmake/projects/glfw/hunter.cmake

```diff
@@ -25,7 +25,7 @@ hunter_pick_scheme(DEFAULT url_sha1_cmake)
 hunter_cacheable(glfw)
 hunter_download(
     PACKAGE_NAME glfw
-    PACKAGE_INTERNAL_DEPS_ID "1"
```

Comment:
This package use standard CMake scheme so PACKAGE_INTERNAL_DEPS_ID should not be changed.